### ISSUE TITLE
Clarify transaction behavior when executing Multiple Queries in TablePlus

### ISSUE THREAD

**Issue by christiaanwesterbeek** [TablePlus/TablePlus#3392]
> I think you're just seeing normal SQL behavior, TablePlus doesn't do anything special here. It just submits all the queries one by one and returns the result.

_Originally posted by @huyphams in https://github.com/TablePlus/TablePlus/issues/1268#issuecomment-506583696_

As a former Postico user, I have been bitten by this twice now. When I would select multiple queries in Postico and execute them, Postico would wrap them in a transaction, so if one would fail, then all queries within the selection would be rolled back. After years of using Postico and getting so used to that behavior, TablePlus caught me by surprise after discovering that a few queries within the selection could fail while others would succeed.

To summarize:

- Postico wraps multiple selected queries in a single transaction by default, ensuring that if one query fails, they all fail.
- TablePlus, on the other hand, executes each query independently unless explicitly wrapped in a transaction.

Postico has [documentation about its behavior](https://eggerapps.at/postico/docs/v1.5.22/sql-query-view.html), but TablePlus has not.
